_config.yml

```diff
@@ -0,0 +1 @@
+exclude: [charts]
```

Comment:
FYI, this is something we included to force Jekyll (which is what GitHub Pages uses to render your site) to drop the charts/ directory from being served. This was done because Jekyll tries to parse templates the exact same way that Helm does with spacebars (e.g. `{{ value }}`), so we encountered build issues in the partner-charts by exposing this field re: https://github.com/rancher/rancher/issues/31021

Reply:
an in-line comment in the source would be a great place to drop this reminder:
```
prevent jekyll form processing charts templates as if they were jekyll templates
```